android/app/src/androidTest/java/com/trustwallet/core/app/blockchains/digitalgold/TestDigitalGoldAddress.kt

```diff
@@ -0,0 +1,33 @@
+// Copyright © 2017-2020 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+package com.trustwallet.core.app.blockchains.digitalgold
+
+import com.trustwallet.core.app.utils.toHex
+import com.trustwallet.core.app.utils.toHexByteArray
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import wallet.core.jni.*
+
+class TestDigitalGoldAddress {
+
+    init {
+        System.loadLibrary("TrustWalletCore")
+    }
+
+    @Test
+    fun testAddress() {
+        // TODO: Check and finalize implementation
+
+        val key = PrivateKey("__PRIVATE_KEY_DATA__".toHexByteArray())
+        val pubkey = key.publicKeyEd25519
+        val address = AnyAddress(pubkey, CoinType.DIGITALGOLD)
+        val expected = AnyAddress("__EXPECTED_RESULT_ADDRESS__", CoinType.DIGITALGOLD)
+
+        assertEquals(pubkey.data().toHex(), "0x__EXPECTED_PUBKEY_DATA__")
```

Comment:
@haael This is unclear - are you planning to follow-up in this or next PR with actual test cases?